app.py

```diff
@@ -21,7 +20,7 @@
 load_dotenv()
 
 # fertiscan storage config vars
-FERTISCAN_SCHEMA = os.getenv("FERTISCAN_SCHEMA", "fertiscan_0.0.8")
+FERTISCAN_SCHEMA = os.getenv("FERTISCAN_SCHEMA", "fertiscan_0.0.11")
```

Comment:
There is a reason why we hide those value in secrets. It lose its purpose to explicit them in the Github code

Reply:
^^

Reply:
Removed the default value in favor of runtime checks.